pkg/apis/postgres-operator.crunchydata.com/v1beta1/postgrescluster_types.go

```diff
@@ -58,6 +59,17 @@ type PostgresClusterSpec struct {
 	// +optional
 	DisableDefaultPodScheduling *bool `json:"disableDefaultPodScheduling,omitempty"`
 
+	// Environment allows PGO to adapt its behavior according to the specific infrastructure
+	// and/or stage of development the PostgresCluster is associated with.  This includes
+	// requiring and/or loosening the requirements for certain components and settings, while
+	// also providing deeper insights, events and status that more closely align with
+	// PostgresCluster's intended use.
+	// Defaults to “production”.  Acceptable values are "development" and "production".
+	// +kubebuilder:validation:Enum={production,development}
+	// +kubebuilder:default=production
+	// +optional
+	Environment *string `json:"environment,omitempty"`
```

Comment:
> If so, will we "always" have a value?

These semantics always trip me up too.  But yea, like you mentioned, the field should/will always have a value, since the API will will in the default if nothing is provided by the user.